src/data_ingestion/fhir_ingestion.py

```
@@ -0,0 +1,313 @@
+    Args:
+        encounter_data: A FHIR Encounter resource as a Python dictionary.
+
+    Returns:
+        A list of dictionaries, where each dictionary is an event record.
+    """
+    events: List[Dict[str, Any]] = []
+    try:
+        encounter = Encounter.model_validate(encounter_data)
+
+        if not encounter.id:
+            print(f"Warning: Encounter resource missing ID. Skipping.")
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)